src/components/House/House.tsx

```diff
@@ -1,56 +1,108 @@
-import {Container} from '../../styles/Container';
 import {Floor} from '../Floor/Floor';
 import {createFloors} from '@src/utils';
 import {Lift} from '@src/components/Lift/Lift';
-import {useState} from 'react';
+import {useEffect, useState} from 'react';
 import {SHouse} from '@src/components/House/styles';
 import {FLOOR_COUNT} from '@src/constants';
+import {RefreshButton} from '@src/components/RefreshButton/RefreshButton';
+import {LiftLocationType} from '@src/components/Lift/types';
+import {Wrapper} from '@src/styles/Wrapper';
 
-export type FloorValueButtonType = {
-    floor: number
-    isPressed: boolean
+type LiftInMotionType = {
+    isLeftLiftMoving: boolean
+    isRightLiftMoving: boolean
 }
 
 export const House = () => {
 
-    const [floorValueButton, setFloorValueButton] = useState<FloorValueButtonType>({floor: 1, isPressed: false});
+        const [queue, setQueue] = useState<number[]>([])
+        const [targetLiftLeft, setTargetLiftLeft] = useState<number>(1)
+        const [targetLiftRight, setTargetLiftRight] = useState<number>(1)
+        const [liftsState, setLiftsState] = useState<LiftInMotionType>({isLeftLiftMoving: false, isRightLiftMoving: false})
 
-    const floors = createFloors(FLOOR_COUNT)
+        const floors = createFloors(FLOOR_COUNT)
 
-    const onStartLift = (floor: number) => {
-        setFloorValueButton(
-            {...floorValueButton, floor: floor, isPressed: true}
-        )
-    }
+        const getIsButtonPressed = (floor: number) => {
+            return (
+                (queue.find(q => q === floor) !== undefined) ||
+                ((liftsState.isRightLiftMoving && targetLiftRight === floor) ||
+                    ((liftsState.isLeftLiftMoving && targetLiftLeft === floor)))
+            )
+        }
```

Comment:
очень неудобно читать... 

```suggestion
        const getIsButtonPressed = (floor: number) => {
        const isFloorInQueue = queue.find(q => q === floor) !== undefined;
        const isRightLiftMovingToFloor = liftsState.isRightLiftMoving && targetLiftRight === floor;
        const isLeftLiftMovingToFloor = liftsState.isLeftLiftMoving && targetLiftLeft === floor;
            return isFloorInQueue  || (isRightLiftMovingToFloor  || isLeftLiftMovingToFloor
        }
```